release_notes.md

```diff
@@ -1,15 +1,15 @@
 # What's New
 
-- The AI Assistant can now understand and respond to context sensitive conversations and perform given SOC tasks.
+**Enhanced AI Assistant with Voice Recognition Feature**
```

Comment:
New release note as that now we are not deleting the thread from OpenAI, we are only clearing the conversation